.storybook/main.ts

```diff
@@ -17,6 +17,16 @@ const config: StorybookConfig = {
       strictMode: true,
     },
   },
+  swc: () => ({
+    jsc: {
+      transform: {
+        react: {
+          // Do not require importing React into scope to use JSX
+          runtime: 'automatic',
+        },
+      },
+    },
+  }),
```

Comment:
Somehow:
- pre-push hooks didn't like unused React imports in some stories (how did folks push to their branches before?)
- Stories without React in scope won't compile